src/corelibs/U2Algorithm/src/util_msa_consensus/MaConsensusAlgorithmSimpleExtended.cpp

```diff
@@ -155,7 +155,8 @@ char MaConsensusAlgorithmSimpleExtended::getConsensusChar(const MultipleAlignmen
     QVector<QVector<char>> frequencies = getFrequences(ma, column, seqIdx);
 
     char bestCharacter = INVALID_CONS_CHAR;
-    const int thresholdCount = qRound(static_cast<double>((frequencies.size() - 1) * getThreshold()) / 100);
+    int thresholdCount = ((frequencies.size() - 1) * getThreshold()) / 100;
+    thresholdCount += (((frequencies.size() - 1) * getThreshold()) % 100) == 0 ? 0 : 1;
```

Comment:
This code is ok, but I offer to use "ceil" function to round to the bigger integer.